ajax/libs/flowplayer/package.json

```diff
@@ -16,7 +16,11 @@
     "type": "git",
     "url": "https://github.com/flowplayer/flowplayer"
   },
-  "license": "GPLv3",
+  "author": {
+    "name": "Niklas Närhinen",
+    "url": "https://github.com/nnarhinen"
+  },
+  "license": "GPL-3.0",
```

Comment:
@hare1039 Please notice the position of the field. You don't need to make `license` location changed.